public_html/wp-content/plugins/wordcamp-payments-network/includes/sponsor-invoices-dashboard.php

```diff
@@ -461,3 +464,107 @@ function delete_index_row( $invoice_id ) {
 		)
 	);
 }
+
+/**
+ * Schedule cron to send reminder mails to organizers for unpaid invoices.
+ */
+function schedule_sent_invoice_reminder() {
+	if ( wp_next_scheduled( 'send_invoice_pending_reminder' ) ) {
+		return;
+	}
+
+	wp_schedule_event( time(), 'daily', 'send_invoice_pending_reminder' );
+}
+
+/**
+ * Send reminder to organizer about the unpaid invoice.
+ */
+function send_invoice_pending_reminder() {
+	global $wpdb;
+
+	$table_name    = get_index_table_name();
+	$sent_invoices = $wpdb->get_results( "
+		SELECT blog_id, invoice_id
+		FROM $table_name
+		WHERE status = 'wcbsi_approved'
+		LIMIT 1000
+	" );
+
+	foreach ( $sent_invoices as $invoice_data ) {
+		$blog_id    = $invoice_data->blog_id;
+		$invoice_id = $invoice_data->invoice_id;
+
+		switch_to_blog( $blog_id );
+
+		$invoice         = get_post( $invoice_id );
+		$invoice_sent_at = get_post_meta( $invoice_id, 'Sent at', true );
+
+		if ( empty( $invoice_sent_at ) ) {
+			// Backfill for older invoices.
+			update_post_meta( $invoice_id, 'Sent at', time() );
+			update_post_meta( $invoice_id, 'Backfilled Sent at', true );
+			restore_current_blog();
+			continue;
+		}
+
+		$last_reminder  = get_post_meta( $invoice_id, 'last_reminder_details', true );
+		$reminder_step  = 1;
+		$last_step_time = $invoice_sent_at;
+		if ( ! empty( $last_reminder ) ) {
+			$reminder_step  = $last_reminder['step'] + 1;
+			$last_step_time = $last_step_time['sent_at'];
+		}
+
+		// We will send reminders after 30, 45, and 60 days.
+		$reminder_schedule = array(
+			1 => 30,
+			2 => 15,
+			3 => 15,
+		);
+
+		if ( $reminder_step > count( $reminder_schedule ) ) {
+			// At this point 2 months have been passed. Maybe send a mail to Community support at this point.
+			restore_current_blog();
+			continue;
+		}
+
+		$next_reminder_in = $last_step_time + $reminder_schedule[ $reminder_step ] * DAY_IN_SECONDS;
+
+		if ( time() < (int) $next_reminder_in ) {
+			restore_current_blog();
+			continue;
+		}
+
+		$current_reminder_details = array(
+			'sent_at' => time(),
+			'step'    => $reminder_step,
+		);
+
+		$edit_link = get_site_url() . "/wp-admin/post.php?post=$invoice_id&action=edit";
+
+		$reminder_body = str_replace(
+			"\t",
+			'',
+			"Howdy organizers,
+					<br>
+					It looks like the invoice <a href='$edit_link'>{$invoice->post_title}</a> is still unpaid. If you still expect the sponsor to pay this invoice, please contact them to find out when we should expect payment. If this invoice needs to be cancelled, please email support@wordcamp.org.
+					<br>
+					Thanks for all your hard work on WordCamp."
+		);
+
+		$author = get_user_by( 'ID', $invoice->post_author );
+		wp_mail(
+			array( $author->user_email ),
```

Comment:
Should the recipient be the invoice creator, or maybe the main email address for the WordCamp? I'm thinking the latter might be better because often the main email address forwards to the whole organizing team.

Reply:
Maybe send it to the invoice creator (since they're primarily responsible), but also CC the lead organizer or main address (in case the invoice creator has flaked out) ?

Reply:
added a cc to lead organizer.